cmd/wallet-web/src/pages/DIDManagement.vue

```diff
@@ -7,9 +7,9 @@
 <template>
   <div class="flex flex-col justify-start items-start w-full">
     <tabs>
-      <tab title="Digital Identity Preference"><preferences :allDIDs.sync="allDIDs" /></tab>
-      <tab title="Create ORB Digital Identity"><create-orb :allDIDs.sync="allDIDs" /></tab>
-      <tab title="Import Any Digital Identity"><import-did :allDIDs.sync="allDIDs" /></tab>
+      <tab title="Digital Identity Preference"><preferences :all-d-i-ds.sync="allDIDs" /></tab>
+      <tab title="Create ORB Digital Identity"><create-orb :all-d-i-ds.sync="allDIDs" /></tab>
+      <tab title="Import Any Digital Identity"><import-did :all-d-i-ds.sync="allDIDs" /></tab>
```

Comment:
Eh...we need to come up with some other naming convention to avoid this stuff...

Reply:
Let's make sure this does not break the DID management page @talwinder50